package.json

```diff
@@ -11,13 +11,14 @@
     "lint": "eslint '**/*.{ts,tsx}' '**/*.d.{ts,tsx}' --cache",
     "lint:fix": "eslint --fix '**/*.{ts,tsx}' --cache",
     "lint:fix-pre-commit": "eslint -c .eslintrc.pre-commit.js --fix '**/*.{ts,tsx}' --no-eslintrc --cache",
-    "prepare": "husky install",
+    "prepare": "husky install && rm -rf dist/ && tsc && webpack && api-extractor run --local",
     "pre-commit": "lint-staged && tsc && yarn docs",
     "typecheck": "tsc",
     "test": "yarn test:unit",
     "test:unit": "NODE_ENV=test jest '.*\\.spec\\.ts'",
-    "docs": "tsc && api-extractor run --local --verbose && api-documenter markdown -i ./temp -o ./docs"
+    "docs": "api-documenter markdown -i ./temp -o ./docs"
   },
+  "jsdelivr": "dist/eppo-sdk.js",
```

Comment:
[jsdelivr](https://www.jsdelivr.com/package/npm/@eppo/js-client-sdk) is the CDN that the SDK is loaded from:
```
<script src="https://cdn.jsdelivr.net/npm/@eppo/js-client-sdk@0.0.5/dist/eppo-sdk.min.js"></script>```